include/revng/Pipes/Kinds.h

```diff
@@ -36,6 +36,8 @@ inline FunctionKind
 
 inline pipeline::Kind
   BinaryCrossRelations("BinaryCrossRelations", &ranks::Binary);
+inline pipeline::Kind CallGraphSVG("CallGraphSVG", &ranks::Binary);
+inline FunctionKind CallGraphSliceSVG("CallGraphSliceSVG", &ranks::Function);
```

Comment:
What's a slice of the call graph?

Reply:
We used to call them `XRef`s in cold-revng.

Reply:
:+1: